CriteoPublisherSdk/Sources/Network/CR_ApiHandler.h

```diff
@@ -71,6 +71,7 @@ typedef void (^CR_LogsCompletionHandler)(NSError *error);
                config:(CR_Config *)config
            deviceInfo:(CR_DeviceInfo *)deviceInfo
               context:(__unused CRContextData *)contextData
+                coppa:(NSNumber *)coppa
```

Comment:
nullable. Also, please keep the original naming until the bid serializer rename the flag to coppa

Reply:
Same issue here, you can't have only one nullability specifier. You need to add them to all the variables.

Reply:
I pushed an update so the original name `childDirectedTreatment` is used until the last step where the `CR_BidRequestSerializer` sends it as `coppa`.